src/components/QuestionPlay/index.tsx

```diff
@@ -1,135 +1,97 @@
-        <Button
-          type="button"
-          color="blue"
-          onClick={() => {
-            setIsAnswerRevealed(true);
-
-            // pause timer when the answer is revealed
-            if (pauseTimer) {
-              pauseTimer();
-            }
-          }}>
-          Reveal answer
-        </Button>
-      )}
-      <Divider />
-      {isAttempted && (!isPreview || isQuestionSaved) ? (
-        <Button size="large" onClick={() => onClose()} type="button" color="blue">
-          Close
-        </Button>
-      ) : isWithoutOptions && !isPreview ? (
-        // Show correct & incorrect button in non preview mode when playing
-        // When answer is revealed by quiz host show correct or incorrect button, clicking on which will add points accordingly
-        isAnswerRevealed && (
-          <div className="flex">
-            <Button type="button" size="large" className="fullWidth" color="red" onClick={() => submitResponse(null)}>
-              Incorrect
-            </Button>
-            <Button
-              type="button"
-              size="large"
-              className="ml-lg fullWidth"
-              color="green"
-              onClick={() => submitResponse(options[0].optionId)}>
-              Correct
-            </Button>
-          </div>
-        )
-      ) : (
-        // for preview mode show save button & while playing show submit button for mcq
-        <Button type="submit" size="large" color="green">
-          {isPreview ? 'Save' : 'Submit'}
-        </Button>
-      )}
-    </form>
+    <Card shadow="sm" p="lg" radius="md" my="sm" withBorder className="secondaryCard">
+      <form onSubmit={handleSubmit(() => submitResponse(selectedChoice === '' ? null : selectedChoice))}>
+        <Group>
+          <Title mr="xl" order={4}>
+            Question {selectedQuestion.questionNum}
+          </Title>
+          <Text weight="bold" component="span" size="sm">
+            Points: {points}
+          </Text>
+        </Group>
+        <Box my="xs">
+          <Markdown>{text}</Markdown>
+        </Box>
+        {isWithoutOptions ? (
+          <WithoutOptions
+            isAnswerRevealed={isAnswerRevealed}
+            setIsPlaying={setIsPlaying}
+            options={options}
+            setIsAnswerRevealed={setIsAnswerRevealed}
+            setSelectedChoice={setSelectedChoice}
+            isAttempted={isAttempted}
+          />
+        ) : (
+          <WithOptions
+            options={options}
+            setSelectedChoice={setSelectedChoice}
+            selectedOptionId={selectedOptionId}
+            selectedChoice={selectedChoice}
+            isAttempted={isAttempted}
+            isPlaying={isPlaying}
+          />
+        )}
+        {isAttempted && !winner && (
+          <Button mt="xl" variant="default" radius="md" onClick={continueGame}>
+            Continue
+          </Button>
+        )}
+        <button className="displayNone" id="btnSubmitResponse" type="submit">
```

Comment:
Why are we using this extra hidden button?
Can't we submit the form directly?

Reply:
@govind007 ?

Reply:
submit the form directly is reloading the page, tried event.preventDefault but it's not working